x-pack/platform/plugins/shared/onechat/server/services/tools/builtin/definitions/get_dataview.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from '@kbn/zod';
+import { platformCoreTools, ToolType } from '@kbn/onechat-common';
+import type { BuiltinToolDefinition } from '@kbn/onechat-server';
+import { ToolResultType } from '@kbn/onechat-common/tools/tool_result';
+
+const getDataViewSchema = z.object({
+  id: z.string().describe('ID of the data view to retrieve.'),
+});
+
+export const getDataViewTool = (): BuiltinToolDefinition<typeof getDataViewSchema> => {
+  return {
+    id: platformCoreTools.getDataView,
+    type: ToolType.builtin,
+    description:
+      'Retrieve detailed information about a specific Kibana data view, including its field mappings and configuration.',
+    schema: getDataViewSchema,
```

Comment:
So what's the exact idea behind those `get_dataview` and `list_dataviews` tools?

Are we expecting the dashboard agent to call `list_dataviews` first before thinking about generating visualizations, so that it can provide the right dataview to use? Or are we going to do that kind of selection more automatically / implicitly inside the visualization creation tool via the `dataview_explorer`?

Trying to figure out how complex the actual dashboard generation flow is going to be / which amount of logic will be in charge of the actual agent.